stan/math/fwd/mat/fun/divide.hpp

```diff
@@ -11,8 +11,8 @@ template <typename T, int R, int C>
 inline Eigen::Matrix<fvar<T>, R, C> divide(
     const Eigen::Matrix<fvar<T>, R, C>& v, const fvar<T>& c) {
   Eigen::Matrix<fvar<T>, R, C> res(v.rows(), v.cols());
-  for (int i = 0; i < v.rows(); i++) {
-    for (int j = 0; j < v.cols(); j++) {
+  for (int j = 0; j < v.cols(); j++) {
```

Comment:
If you wouldn't mind taking one more pass, a simpler and more efficient way to do this that will work for matrices, vectors, and row vectors is:

```
for (int i = 0; i < v.size(); ++i)
  res(i) = v(i) / c;
```

Reply:
Yep that will work. Thank you. I'm going to make this change a just a bit.